examples/complete/grants.us-east-2.tfvars

```diff
@@ -10,10 +10,16 @@ acl = ""
 
 grants = [
   {
-    id          = null
-    type        = "Group"
-    permissions = ["READ", "WRITE"]
```

Comment:
why is grant permissions being split from an array to individual strings here [docs](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket#permissions)

Reply:
@Benbentwo you're pointing to s3-bucket instead of the aws_s3_bucket_acl

https://registry.terraform.io/providers/hashicorp/aws/latest/docs/guides/version-4-upgrade#grant-argument